03LessonThree/in-class-project/daily-planet.html

```diff
@@ -5,6 +5,39 @@
         <link rel="stylesheet" href="css/daily-planet-style.css"/>
     </head>
     <body>
-        <!-- You code start here -->
+        <header>
+
+          <h1>Daily Planet</h1>
+
+          <nav>
+            <a href="http://www.google.com">Sports</a>
+            <a href="http://www.google.com">Local</a>
+            <a href="http://www,google.com">Economy</a>
+          </nav>
+
+        </header>
+
+        <main>
+
+          <section>
+
+            <article>
+              <h3>Sanders Holds Early Lead in West Virginia</h3>
+              <a href="http://www.cnn.com/2016/05/10/politics/west-virginia-nebraska-primary-highlights/"><img src="http://www.motherjones.com/files/bernie_2.jpg"></a>
+            </article>
+
+          </section>
+
+          <aside>
+            <h4>Our Beautiful Sponsers</h4>
+            <img src="https://pbs.twimg.com/profile_images/646380034068889601/pjffPFT5.png">
+          </aside>
+
+        <footer>
+          <p>Copyright 2016</p>
+        </footer>
+
+        </main>

```

Comment:
I believe the main didn't encapsulate the footer, but outside of this, great job! looks organized and clean. I would start commenting your code though.